apps/supernova/docker/VERSION

```diff
@@ -1 +1 @@
-1.0.0
\ No newline at end of file
+1.1.0
```

Comment:
new line

Reply:
I wouldn't do it since the version is read from the pipeline and I want to avoid new lines chars or similar...

Reply:
the version to deploy is being read with `cat ${{ env.PACKAGE_PATH }}/docker/VERSION)` and I want to avoid to set new lines in the version for the image

Reply:
please extend your command:

```bash
cat ${{ env.PACKAGE_PATH }}/docker/VERSION | tr -d '\n'
```